components/automate-cli/cmd/chef-automate/initConfigHaTmpl.go

```diff
@@ -359,6 +363,20 @@ chef_server_private_ips = []
 opensearch_private_ips = []
 postgresql_private_ips = []
 ## === ===
+
+
+## Object storage similar to AWS S3 Bucket
+## 
+[object_storage.config]
```

Comment:
we can add [object_storage.config] next to the backup_config

Reply:
Done